admin/bin/qserv-admin.py

```diff
@@ -154,7 +186,7 @@ def receiveCommands(self):
         cmd = ''
         prompt = self._prompt
         while True:
-            line = raw_input(prompt).decode("utf-8").strip()
+            line = raw_input(prompt).strip()
```

Comment:
I don't think this line or its previous version is correct in Python 2 (consider what happens when grabbing input in a non-ASCII compatible encoding). I'm not that experienced with string encoding, but I think it might be more correct to say something like:
```
line = raw_input().decode(sys.stdin.encoding or locale.getpreferredencoding()).encode('utf-8')
```